internal/connector/pingone_platform/pingone_platform_connector.go

```diff
@@ -0,0 +1,109 @@
+package pingone_platform
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/patrickcping/pingone-go-sdk-v2/pingone"
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/connector/common"
+	"github.com/pingidentity/pingctl/internal/connector/pingone_platform/resources"
+	"github.com/pingidentity/pingctl/internal/logger"
+)
+
+const (
+	serviceName = "pingone-platform"
+)
+
+// Verify that the connector satisfies the expected interfaces
+var (
+	_ connector.Exportable      = &PingonePlatformConnector{}
+	_ connector.Authenticatable = &PingonePlatformConnector{}
+)
+
+type PingonePlatformConnector struct {
+	clientInfo connector.SDKClientInfo
+}
+
+// Utility method for creating a PingonePlatformConnector
+func PlatformConnector(ctx context.Context, apiClient *sdk.Client, exportEnvironmentID string) *PingonePlatformConnector {
+	return &PingonePlatformConnector{
+		clientInfo: connector.SDKClientInfo{
+			Context:             ctx,
+			ApiClient:           apiClient,
+			ExportEnvironmentID: exportEnvironmentID,
+		},
+	}
+}
+
+func (c *PingonePlatformConnector) Export(format, outputDir string, overwriteExport bool) error {
+	l := logger.Get()
+
+	l.Debug().Msgf("Validating export environment ID...")
+
+	environment, response, err := c.clientInfo.ApiClient.ManagementAPIClient.EnvironmentsApi.ReadOneEnvironment(c.clientInfo.Context, c.clientInfo.ExportEnvironmentID).Execute()
```

Comment:
Look into moving this into common code so each connector doesn't do this. Perhaps move it out to export.go?

Reply:
We could always do it later as well. It's not a ton of repeatable code.